phpseclib/System/SSH/Common/Traits/ReadBytes.php

```diff
@@ -23,10 +25,9 @@ trait ReadBytes
     /**
      * Read data
      *
-     * @param int $length
      * @throws \RuntimeException on connection errors
      */
-    public function readBytes($length)
+    public function readBytes(int $length)
```

Comment:
We should prob add a return type hint of string to this? If `fread` returns anything other than a string an exception prob ought to be thrown